scripts/docker-entrypoint.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+if [ -z "${SOURCECRED_GITHUB_TOKEN}" ]; then
+    echo "Export SOURCECRED_GITHUB_TOKEN with the container."
+    echo "docker run --env SOURCECRED_GITHUB_TOKEN=$(cat .token) -p 8080:8080 sourcecred <repository>"
+    exit 1;
+fi
+
+# serves on 8080 in docker container, should be bound to host
+node /code/bin/sourcecred.js load "${@}"
+yarn start --host 0.0.0.0
```

Comment:
From the top of my head, this call may also need to be run with `exec` in order to pass on signals.

Reply:
This would make sense - when I tried without detached it hung and I couldn't control C. Can you show me an example of what it should be (`yarn exec --host 0.0.0.0`) ?

Reply:
It's a built-in sh/bash function and it goes in front of your command.

```sh
exec yarn --host 0.0.0.0
```

```
$ help exec
exec: exec [-cl] [-a name] [command [arguments ...]] [redirection ...]
    Replace the shell with the given command.
    
    Execute COMMAND, replacing this shell with the specified program.
    ARGUMENTS become the arguments to COMMAND.  If COMMAND is not specified,
    any redirections take effect in the current shell.
    
    Options:
      -a name	pass NAME as the zeroth argument to COMMAND
      -c	execute COMMAND with an empty environment
      -l	place a dash in the zeroth argument to COMMAND
    
    If the command cannot be executed, a non-interactive shell exits, unless
    the shell option `execfail' is set.
    
    Exit Status:
    Returns success unless COMMAND is not found or a redirection error occurs.

```

Reply:
Oh of course - you want to pass execution off to yarn. Sure let me try that now, it would be better usability to be able to run and kill with control C than in detached.

Reply:
exec doesn't work, so I'll be happy to add it, but ti doesn't allow control C :(

I'm testing the changes you just pushed now!

Reply:
SO MUCH FASTER! :rocket:  And no wacky error messages. Dinosaur approved!